lib/l10n/messages_all_locales.dart

```diff
@@ -13,49 +13,37 @@ import 'package:intl/intl.dart';
 import 'package:intl/message_lookup_by_library.dart';
 import 'package:intl/src/intl_helpers.dart';
 
-import 'messages_de.dart' as messages_de;
 import 'messages_en.dart' as messages_en;
-import 'messages_es.dart' as messages_es;
+import 'messages_de.dart' as messages_de;
 import 'messages_gl.dart' as messages_gl;
-import 'messages_it.dart' as messages_it;
-import 'messages_messages.dart' as messages_messages;
 import 'messages_nl.dart' as messages_nl;
-import 'messages_ru.dart' as messages_ru;
 import 'messages_vi.dart' as messages_vi;
+import 'messages_ru.dart' as messages_ru;
 
 typedef Future<dynamic> LibraryLoader();
 Map<String, LibraryLoader> _deferredLibraries = {
```

Comment:
I'm not sure what has happened here, but you've deleted some of the support language. It is really important you use your IDE to review all the changes you have made and ensure you are happy with them.